sdks/java/io/elasticsearch/pom.xml

```diff
@@ -133,6 +123,12 @@
 
     <dependency>
       <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
```

Comment:
same as above, this is only used in the tests not in the IO, and I think if the developer of the IO is going to use it, he can change the scope once needed.